src/index.html

```diff
@@ -18,5 +29,85 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo__image"
+        href="index.html"
+      >
+        <img
+          src="images/moyo_1.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
```

Comment:
Add class to tag li everywhere
```suggestion
          <li class="nav__item">
```